funnel/templates/project_registered_email.html.jinja2

```diff
@@ -0,0 +1,7 @@
+Hi {{ current_auth.user.fullname.split()[0] }},
```

Comment:
It's better if you can add a property called `first_name` in User models and do this there. Or just use fullname.

Reply:
This should remain `fullname`. Do not assume the first part is the given name.